src/modules/dashboard/components/Extensions/ExtensionDetails.css

```diff
@@ -4,6 +4,10 @@
   grid-template-columns: minmax(460px, 2fr) minmax(340px, 1fr);
 }
 
+.main a {
```

Comment:
I think this selector is making the explorer links look like this:

![FireShot Capture 432 - Colony - localhost](https://user-images.githubusercontent.com/18473896/132423807-b7f7e3f3-0daa-479c-8f6a-add33163ab8b.png)

Reply:
good catch - fixed now.